client/src/components/FormAction/FormAction.js

```diff
@@ -149,10 +192,15 @@ FormAction.propTypes = {
   name: React.PropTypes.string,
   handleClick: React.PropTypes.func,
   title: React.PropTypes.string,
+  // Action text when there's no changes detected in the form
+  savedTitle: React.PropTypes.string,
+  savedIcon: React.PropTypes.string,
+  savedClasses: React.PropTypes.string,
   type: React.PropTypes.string,
   loading: React.PropTypes.bool,
   icon: React.PropTypes.string,
   disabled: React.PropTypes.bool,
+  changed: React.PropTypes.bool,
```

Comment:
this should be named `formChanged` I think, not to be confused with this actual component changing somehow